db/app.js

```diff
@@ -15,20 +18,29 @@ app.get("/api/articles/:article_id", getArticleById)
 
 app.get("/api/articles/:article_id/comments", getCommentByArticleId)
 
+app.post("/api/articles/:article_id/comments", postCommentWithId)
+
 app.get("/api/articles", getAllArticles)
 
 
 app.use((err, req, res, next) => {
```

Comment:
Might be a good time to extract these error handlers out into their own file to keep the app.js nice and tidy